server/s3_put_multiobject_action.cc

```diff
@@ -310,6 +310,14 @@ void S3PutMultiObjectAction::fetch_firstpart_info_failed() {
   send_response_to_s3_client();
 }
 
+void S3PutMultiObjectAction::_set_layout_id(int layout_id) {
+  assert(layout_id > 0 && layout_id < 15);
```

Comment:
max layout id is 9 I feel

Reply:
I inserted these asserts for something like uninitialized variable or other rude errors.